test/integration/physics_system.cc

```diff
@@ -2196,3 +2196,68 @@ TEST_F(PhysicsSystemFixture,
     EXPECT_NEAR(0.0, wrench.torque().z(), 1e-3);
   }
 }
+
+/////////////////////////////////////////////////
+// Test that joint velocity limit is applied
+TEST_F(PhysicsSystemFixtureWithDart6_10,
+    IGN_UTILS_TEST_DISABLED_ON_WIN32(JointVelocityLimitTest))
+{
+  // Start server
+  ServerConfig serverConfig;
+  const auto sdfFile = std::string(PROJECT_SOURCE_PATH) +
+          "/test/worlds/joint_velocity_limit.sdf";
+  serverConfig.SetSdfFile(sdfFile);
+
+  auto server = std::make_unique<Server>(serverConfig);
+  EXPECT_FALSE(server->Running());
+  EXPECT_FALSE(*server->Running(0));
+
+  using namespace std::chrono_literals;
+  server->SetUpdatePeriod(1ns);
+
+  test::Relay testSystem;
+
+  const std::size_t iterTestStart{100};
+  const std::size_t nIters{500};
+  testSystem.OnPreUpdate(
+      [&](const gazebo::UpdateInfo &_info, gazebo::EntityComponentManager &_ecm)
+      {
+        // Create components, if the don't exist, on the first iteration
+        if (_info.iterations == 1)
+        {
+          for (const auto &e : _ecm.EntitiesByComponents(components::Joint()))
+          {
+            if (!_ecm.Component<components::JointVelocity>(e))
+            {
+              _ecm.CreateComponent(e, components::JointVelocity());
+            }
+          }
+        }
+      });
+
+  testSystem.OnPostUpdate(
+      [&](const gazebo::UpdateInfo &_info,
+          const gazebo::EntityComponentManager &_ecm)
+      {
+        // After nIters iterations, check angular velocity of each of the joints
+        if (_info.iterations == iterTestStart + nIters)
+        {
+          int count = 0;
+          for (const auto &e : _ecm.EntitiesByComponents(components::Joint()))
+          {
+            auto *jointVel = _ecm.Component<components::JointVelocity>(e);
+            EXPECT_NE(nullptr, jointVel);
+            EXPECT_FALSE(jointVel->Data().empty());
+            if (jointVel->Data().size() > 0)
+            {
+              ++count;
+              EXPECT_LT(std::abs(jointVel->Data()[0]), 1.1);
```

Comment:
Since the velocity limit in `joint_velocity_limit.sdf` is set to `1`, shouldn't we check `<= 1` instead? Also, I don't think `std::abs` is necessary
```suggestion
              EXPECT_LE(jointVel->Data()[0], 1.0);
```

Reply:
Done https://github.com/ignitionrobotics/ign-gazebo/pull/1445/commits/0710aabf28d03a32a0d9b65e0662b4c4d8abb6e6